app.js

```diff
@@ -1,46 +1,14 @@
 const express = require('express');
 const app = express();
-const got = require('got');
+const swapi = require('./api/swapi.js');
 
 app.get('/:id',async (req, res) => {
-    const url = `https://swapi.co/api/people/${req.params.id}`;
     try{
-         const character = await getData(url);
-         const homeWorld = await getData(character.body.homeworld);
-         const species = await getData(character.body.species[0]);
-         const films = await getFilms(character.body.films);
-         res.status(200).send({...character.body,homeworld:homeWorld.body,species:species.body,films:films})
+         const detailedCharacter = await swapi.getDetailedCharacter(req.params.id);
+         res.status(200).send(detailedCharacter);
```

Comment:
```suggestion
        return  res.status(200).send(detailedCharacter);
```

Always do a `return` when sending your HTTP response. This prevents code later on attempting to execute, which could potentially cause modifications to an HTTP response that has already been sent. In Express this would cause that app to crash